### PR TITLE
Documentation of the rotational motion of reference frames

### PR DESCRIPTION
**Pull request by pleroy**
#3358.

### CODE REVIEW

File: documentation/Rotational Motion.tex [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+\noindent
+This document describes the computations that are performed by the class \texttt{RigidReferenceFrame} and its subclasses to determine the rotational motion (rotation,
+angular velocity, and angular accelation) of a rigid frame.
+\end{sloppypar}
+
+\section*{Definitions}
+We consider in this document a rigid reference frame defined by two bodies $B_1$ and $B_2$ at positions $\vq_1$ and $\vq_2$, respectively.  We construct a basis of the reference frame using three vectors having the following properties:
+\begin{itemize}
+\item{the \emph{fore} vector $\vF$ which is along the axis $\vq_2 - \vq_1$;}
+\item{the \emph{normal} vector $\vN$ which is orthogonal to $\vF$ and is such that the velocity of the frame, $\dot{\vq}_2 - \dot{\vq}_1$, is in the plane $\pa{\vF, \vN}$;}
```
Comment (eggrobin) [inline documentation/Rotational Motion.tex:31]: Remove `of the frame,`; the velocity of the frame sounds like (𝒒&#x0307;₁ + 𝒒&#x0307;₂) / 2.

Comment (pleroy) [reply documentation/Rotational Motion.tex]: Done.

File: documentation/Rotational Motion.tex [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+\begin{dcases}
+\dot{\vF} &= \dot{\vr} \\
+\dot{\vB} &= \vr \wedge \ddot{\vr} \\
+\dot{\vN} &= \dot{\vB} \wedge \vF + \vB \wedge \dot{\vF}
+\end{dcases}
+\label{eqndFBN}
+\end{equation}
+Injecting these expressions in the derivative formula (\ref{eqndv}) makes it possible to compute the trihedron of the derivatives $\pa{\dot{\vf}, \dot{\vn}, \dot{\vb}}$ of (\ref{eqnfbn}).  The angular velocity is then written:
+\[
+\VectorSymbol{\gw} = \pascal{\scal{\dot{\vn}}{\vb}} \vf + \pascal{\scal{\dot{\vb}}{\vf}} \vn + \pascal{\scal{\dot{\vf}}{\vn}} \vb
```
Comment (eggrobin) [inline documentation/Rotational Motion.tex:95]: Document the derivation of 𝝎.

File: documentation/Rotational Motion.tex [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+\begin{dcases}
+\ddot{\vF} &= \ddot{\vr} \\
+\ddot{\vB} &= \dot{\vr} \wedge \ddot{\vr} + \vr \wedge \vr^{(3)} \\
+\ddot{\vN} &= \ddot{\vB} \wedge \vF + 2 \dot{\vB} \wedge \dot{\vF} + \vB \wedge \ddot{\vF} \\
+\end{dcases}
+\label{eqnddFBN}
+\end{equation}
+Injecting these expressions in the second derivative formula (\ref{eqnddv}) makes it possible to compute the trihedron of the second derivatives $\pa{\ddot{\vf}, \ddot{\vn}, \ddot{\vb}}$ of (\ref{eqnfbn}).  The angular acceleration is then written:
+\begin{align*}
+\dot{\VectorSymbol{\gw}} &= \pascal{\scal{\ddot{\vn}}{\vb}} \vf + \pascal{\scal{\dot{\vn}}{\dot{\vb}}} \vf + \pascal{\scal{\dot{\vn}}{\vb}} \dot{\vf} \\
```
Comment (eggrobin) [inline documentation/Rotational Motion.tex:110]: Document the derivation of 𝝎̇.

File: documentation/Rotational Motion.tex [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+&+ \pascal{\scal{\ddot{\vb}}{\vf}} \vn + \pascal{\scal{\dot{\vb}}{\dot{\vf}}} \vn + \pascal{\scal{\dot{\vb}}{\vf}} \dot{\vn} \\
+&+ \pascal{\scal{\ddot{\vf}}{\vn}} \vb + \pascal{\scal{\dot{\vf}}{\dot{\vn}}} \vb + \pascal{\scal{\dot{\vf}}{\vn}} \dot{\vb}
+\end{align*}
+
+\section*{Jerk}
```
Comment (eggrobin) [inline documentation/Rotational Motion.tex:115]: This is wrong.

**Review by eggrobin — CHANGES_REQUESTED** [review]

File: documentation/Rotational Motion.tex (hunk 4 above)
Comment (pleroy) [reply documentation/Rotational Motion.tex:115]: Fixed.

File: documentation/Rotational Motion.tex (hunk 3 above)
Comment (pleroy) [reply documentation/Rotational Motion.tex:110]: The derivation of 𝝎̇ is rather obvious once the formula for 𝝎 is known.  It's just the formula for the derivative of products.

File: documentation/Rotational Motion.tex (hunk 2 above)
Comment (pleroy) [reply documentation/Rotational Motion.tex:119]: Done.

File: documentation/Rotational Motion.tex [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+\[
+\VectorSymbol{\gG}\of{\vq, \vq_k\of{t}} = \sum{k = 1}[n] \frac{\gm_k \pa{\vq_k - \vq}}{\norm{\vq_k - \vq}^3}
+\]
+The jerk vector field is the total derivative of this field with respect to time:
+\[
+\deriv{t}{\VectorSymbol{\gG}} =
+   \pderiv{\vq}{\VectorSymbol{\gG}} \deriv{t}{\vq} +
+   \sum{k = 1}[n] \pderiv{\vq_k}{\VectorSymbol{\gG}} \deriv{t}{\vq_k} =
+   \pderiv{\vq}{\VectorSymbol{\gG}} \dot{\vq} +
+   \sum{k = 1}[n] \pderiv{\vq_k}{\VectorSymbol{\gG}} \dot{\vq_k}
```
Comment (eggrobin) [inline documentation/Rotational Motion.tex:152]: Put the dot over the q, sticking out over the index: `\dot{\vq}_k`

**Review by eggrobin — APPROVED** [review]

**Pull request merged**